pymeasure/instruments/lecroy/lecroyT3DSO1204.py

```diff
@@ -0,0 +1,829 @@
+        • ac    — AC coupling block DC component in the trigger path, removing dc offset voltage from the trigger 
+                  waveform. Use AC coupling to get a stable edge trigger when your waveform has a large dc offset.
+        • dc    — DC coupling allows dc and ac signals into the trigger path.
+        • lowpass  — HFREJ coupling places a lowpass filter in the trigger path.
+        • highpass — LFREJ coupling places a highpass filter in the trigger path.
+        """,
+        validator=strict_discrete_set,
+        values={"ac": "AC", "dc": "DC", "lowpass": "HFREJ", "highpass": "LFREJ"},
+        map_values=True
+    )
+
+    trigger_level = Instrument.control(
+        "TRLV?", "TRLV %eV",
+        """ A float parameter that sets the trigger level voltage for the active trigger source. 
+            When there are two trigger levels to set, this command is used to set the higher trigger 
+            level voltage for the specified source. TRLV2 is used to set the lower trigger level voltage.
+            The trigger level is -4.5*DIV to 4.5*DIV.
+            An out-of-range value will be adjusted to the closest legal value.
+        """
+        # TODO dynamic range
+    )
+
+    trigger_level2 = Instrument.control(
+        "TRLV2?", "TRLV2 %eV",
+        """ A float parameter that sets the lower trigger level voltage for the specified source.
+        Higher and lower trigger levels are used with runt/slope triggers.
+        The trigger level is -4.5*DIV to 4.5*DIV.
+        An out-of-range value will be adjusted to the closest legal value.
+        """
+        # TODO dynamic range
+    )
+
+    trigger_slope = Instrument.control(
+        "TRSL?", "TRSL %s",
+        """ A string parameter that sets the trigger slope of the specified trigger source.
+        <trig_slope>:={NEG,POS,WINDOW} for edge trigger.
+        <trig_slope>:={NEG,POS} for other trigger
+        """,
+        validator=strict_discrete_set,
+        values={"negative": "NEG", "positive": "POS", "window": "WINDOW"},
+        map_values=True
+    )
+
+    def __init__(self, instrument, number):
+        self.instrument = instrument
+        self.number = number
+
+    def values(self, command, **kwargs):
+        """ Reads a set of values from the instrument through the adapter,
+        passing on any key-word arguments.
+        """
+        return self.instrument.values("C%d:%s" % (self.number, command), **kwargs)
+
+    def ask(self, command):
+        return self.instrument.ask("C%d:%s" % (self.number, command))
+
+    def write(self, command):
+        # only in case of the BWL command the syntax is different. Why? SIGLENT Why?
+        if command == "BWL OFF":
+            self.instrument.write("BWL C%d,OFF" % self.number)
+        elif command == "BWL ON":
+            self.instrument.write("BWL C%d,ON" % self.number)
+        self.instrument.write("C%d:%s" % (self.number, command))
```

Comment:
Is there an `else` missing?

Reply:
Fixed in 795664353b97bf6e4db2098804f57a0e398c43a1